EnglishTranslation-master/SKILL.tsv

```diff
@@ -4809,12 +4809,12 @@ SKILL_20150717_004808	Increases the AoE attack ratio of [Claymore] by 1 per attr
 SKILL_20150717_004809	When an enemy launched by [Punji Stake] hits the ground, it and nearby enemies will be dealt 50% of physical attack as damage.
 SKILL_20150717_004810	Decreases the physical defense of an enemy attacked with [Retrieve] by 10 per attribute level.
 SKILL_20150717_004811	Provides immunity to [Poison] for 2 seconds per attribute level to a character whose [Poison] has been cured due to [Detoxify].
-SKILL_20150717_004812	Even if an object gets out of the range of [Throw Gu Pot], [Poison] will continue for 2 seconds per attribute level.
+SKILL_20150717_004812	Even if an enemy steps out of [Throw Gu Pot], [Poison] will last for 2 seconds per attribute level.
```

Comment:
I commented on this on your first revision:

Maintains [Poison] by 2 seconds per attribute level when enemies are outside the range of [Throw Gu Pot].

or is this better?

Sustains [Poison] by 2 seconds per attribute level when enemies are outside the range of [Throw Gu Pot].